.github/workflows/cmake.yml

```
@@ -33,6 +33,21 @@ jobs:
         path: |
           build/libutf8proc.*
           build/Debug/utf8proc.*
+    - name: Test Consuming (Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
```

* We don't need `${{ ... }}` in `if:`.
*  `runner.os` is better than `matrix.os` because we may use other `matrix.os` such as `windows-11-arm` in the future.
  * See also https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#runner-context for `runner.os`.

```suggestion
      if: runner.os = 'Windows'
```